app/src/org/commcare/dalvik/activities/LoginActivity.java

```diff
@@ -96,9 +97,6 @@
     
     @UiElement(R.id.screen_login_banner_pane)
     private View banner;
-    
-    @UiElement(R.id.str_version)
-    private TextView versionDisplay;

```

Comment:
Why is this getting removed? It looks like it doesn't currently show up on the login screen anyway, but do we want to fix that?